bsp/esp32_p4_function_ev_board/include/bsp/esp32_p4_function_ev_board.h

```diff
@@ -218,7 +226,39 @@ esp_err_t bsp_spiffs_unmount(void);
  * \endcode
  **************************************************************************************************/
 #define BSP_SD_MOUNT_POINT      CONFIG_BSP_SD_MOUNT_POINT
-extern sdmmc_card_t *bsp_sdcard;
+
+/* SD card is connected to Slot 0 pins. Slot 0 uses IO MUX, so not specifying the pins here */
+#define BSP_SDCARD_MMCSLOT_DEFAULT() {\
+    .cd = SDMMC_SLOT_NO_CD,\
+    .wp = SDMMC_SLOT_NO_WP,\
+    .width = 4,\
+    .flags = 0,\
+}
```

Comment:
What about defining a function like `bsp_get_sdmmc_slot_config(slot, &config)`, which would fill the `sdmmc_slot_config_t` structure? This would be future-proof in case a board has two SD card slots.

Reply:
Thank you.
OK, I defined macros due to memory. We can use static variables inside these functions, which will be in memory only after call the function. I will change it.

Reply:
@igrr I updated it with your ideas.